### PR TITLE
fix(main/nushell): use `--features=dataframe`

### PR DESCRIPTION
**Pull request by stevenxxiu**
See https://www.nushell.sh/blog/2022-11-29-nushell-0.72.html.

### CODE REVIEW

**Review by thunder-coding — CHANGES_REQUESTED** [review]
Hey, thanks for the PR. Looks great, just a small one-line change requested. And thanks for going through the nushell changelog and noticing that!

File: packages/nushell/build.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@ TERMUX_PKG_SHA256=cbabc26ce509c672db943f3ac6d44d1e3efbafc3b5929411ac209afe14848e
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="openssl, zlib"
 TERMUX_PKG_BUILD_IN_SRC=true
-TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--features=extra"
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--features=dataframe"
```
[the remark this answers is not in the document]
Comment (stevenxxiu) [reply packages/nushell/build.sh:13]: Thanks, done.

**Review by thunder-coding — APPROVED** [review]

**Comment by stevenxxiu** [comment]
Hmm the build error appears to be due to https://github.com/simd-lite/simd-json/issues/263. Should I just skip the extra features for `arm`?

**Comment by stevenxxiu** [comment]
I skipped the extra features for `arm` before, so it should build. Anything else I need to do?

**Comment by landfillbaby** [comment]
reenabled `--features=extra`. see https://www.nushell.sh/blog/2023-06-27-nushell_0_82.html#continuing-cratification-effort

**Pull request merged**